src/main.rs

```diff
@@ -230,6 +230,21 @@ async fn main() -> Result<()> {
                     .await?;
                 "".to_string()
             }
+            #[cfg(feature = "plugin-vc-zkp-bbs")]
+            ("create_self_issued_credential", Some(sub_m)) => {
+                get_vade_evan(sub_m)?
+                    .helper_create_self_issued_credential(
+                        get_argument_value(sub_m, "schema_did", None),
+                        get_argument_value(sub_m, "credential_subject", None),
+                        get_argument_value(sub_m, "bbs_secret", None),
+                        get_argument_value(sub_m, "bbs_private_key", None),
```

Comment:
Please check if the new arguments are added to match list in => 
fn get_clap_argument(arg_name: &str) -> Result<Arg>

Reply:
Done, thanks.